security-command-center/snippets/src/main/java/management/api/GetSecurityHealthAnalyticsCustomModule.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package management.api;
+
+// [START securitycenter_management_api_get_security_health_analytics_custom_module]
+
+import com.google.cloud.securitycentermanagement.v1.GetSecurityHealthAnalyticsCustomModuleRequest;
+import com.google.cloud.securitycentermanagement.v1.SecurityCenterManagementClient;
+import com.google.cloud.securitycentermanagement.v1.SecurityHealthAnalyticsCustomModule;
+import java.io.IOException;
+
+public class GetSecurityHealthAnalyticsCustomModule {
+
+  public static void main(String[] args) throws IOException {
+
+    // parent: Use any one of the following options:
+    // - organizations/{organization_id}/locations/{location_id}
+    // - folders/{folder_id}/locations/{location_id}
+    // - projects/{project_id}/locations/{location_id}
+    String parent = String.format("organizations/%s/locations/%s", "organization_id", "global");
```

Comment:
ditto regarding formatting input arguments and using org instead of project

Reply:
Addressed